src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -277,21 +280,36 @@ Here's an example of adding the `crossorigin="anonymous"` attribute:
 
 ## Manually record session replays [#manual-replays]
 
-If needed, you can programmatically invoke a session replay using API methods. Here are some use cases:
+If needed, you can programmatically `force` a session replay to record or stop recording using API methods. Here are some use cases:
 
 * Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?":
-    1. Upon user consent, call `NREM.startSessionReplay` to initiate recording. 
-    2. Once the session is complete or consent is withdrawn, use `NREM.stopSessionReplay` to stop capturing data.
+    1. Upon user consent, call `newrelic.recordReplay()` to initiate recording. 
+    2. Once the session is complete or consent is withdrawn, use `newrelic.pauseReplay()` to stop capturing data.
 * Record sessions based on specific criteria. For example:
     * Start recording only for sessions on specific URLs such as a product checkout flow.
     * Stop recording for sensitive pages such as `/billing` or `/health-records`.
 
 See the API documentation for instructions:
 
-* [`NREM.startSessionReplay`](/docs/browser/new-relic-browser/browser-apis/start/)
-* [`NREM.stopSessionReplay`](/docs/browser/new-relic-browser/browser-apis/stop/)
+* [`newrelic.recordReplay`](/docs/browser/new-relic-browser/browser-apis/recordReplay/)
+* [`newrelic.pauseReplay`](/docs/browser/new-relic-browser/browser-apis/pauseReplay/)
 
-To ensure all user interactions are captured during manual recording, set sampling rates to 0% in your application settings.
+## Prevent session replays from automatically tracking [#auto-replays]
+
+If needed, session replays can be prevented from automatically starting, `even when sampled`.  Here are some use cases:
+
+* Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?":
+    1. Upon user consent, call `newrelic.start()` to import the session replay recording module.
+* Record sessions based on specific criteria. For example:
+    * Start recording only for sessions on specific URLs such as a product checkout flow.
```

Comment:
@metal-messiah Can you explain what "prevent replays from automatically tracking/starting" means? Also, the use cases are the same as the section above, so perhaps alternative examples would help clarify what this section's purpose is.

Reply:
@metal-messiah When would you use `newrelic.start` vs. `newrelic.record`?

Reply:
> Can you explain what "prevent replays from automatically tracking/starting" means? Also, the use cases are the same as the section above, so can you provide other examples to help explain what this section is about?

`autoStart: false` + `newrelic.start()` makes it so the agent doesnt automatically do anything when it loads.  It runs in a "deferred" state until `.start()` is called.  Running with autoStart: false and using the `.start()` api ensures that the agent wont create replays at all, even if the backend samples the app to record, until the application calls `.start()

> the use cases are the same as the section above
> When would you use newrelic.start vs. newrelic.record?

The use cases are the same because both mechanisms are useful for those situations, though they have different reasons for those scenarios. 
* You would use `.start` in cases where you dont want `automatic` actions, such as sampling at runtime, to take precedence over some sort of validation, for example -> don't run replay feature on my `HIPAA` page until user consents (and app calls `.start()`)
* You would use `.record` in cases where you want to force-record a replay, even if the sampling was false, for example --> If user gets to the checkout page, i `always` want to record a session replay, even if sampling was false (`.recordReplay`).
other example --> If a user gets to the `HIPAA` page mentioned above, i could manually call `.pauseReplay` to guarantee any running replays get terminated.